### ISSUE TITLE
Members locked out forever

### ISSUE THREAD

**Issue by keithswright** [umbraco/Umbraco-CMS#12985]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

10.0.0

### Bug summary

When a member is locked out after entering their password wrong too many times, they are not allowed try their password again for 30 days.
The default value, I believe is five minutes.
There is a comment in the code mentioning that it has not been implemented yet. 
It is a feature I could use at the moment, cause, what I have currently done is hiked up the number of allowed incorrect login attempts to a high number, to work around the issue.

### Specifics

in `ConfigureMemberIdentityOptions`, we have the following code:

```csharp
        // TODO: Implement this
        options.Lockout.DefaultLockoutTimeSpan = TimeSpan.FromDays(30);
```

### Steps to reproduce

Enter a member password wrong too manty times. 

### Expected result / actual result

There is no way of auto unlocking the members account after a few minutes to let them try again.

---
_This item has been added to our backlog [AB#24980](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/24980)_

**Comment by kjac** [comment]
Hi @keithswright - sorry for the late response 🙄  

You are absolutely correct, this needs implementing. The same also applies to users in  `ConfigureBackOfficeIdentityOptions` 🙈 

```csharp
        // TODO: Implement this
        options.Lockout.DefaultLockoutTimeSpan = TimeSpan.FromDays(30);
```

I reckon both should be configurable from app settings, the only real question is how to approach the default value. To avoid breakage it should remain as 30 days... however, as you point out, the framework default is 5 minutes. Quite the gap!

We will discuss this internally and either add it to an upcoming sprint or make this up for grabs 👍

**Comment by kjac** [comment]
PR in #13808

**Comment by nikolajlauridsen** [comment]
Fixed in #13808, thanks for reporting 😄
